h_docs/polybft_setup.md

```diff
@@ -41,7 +41,7 @@ We need to set native token to be mintable, so we can premine balances to differ
 4. Run the chain
 
 ```
-./hydra server --data-dir ./test-chain-1 --chain genesis.json --grpc-address :5001 --libp2p :30301 --jsonrpc :10001 --log-level DEBUG --log-to ./log
+./hydra server --data-dir ./test-chain-1 --chain genesis.json --secrets-config ./secretsManagerConfig.json --grpc-address :5001 --libp2p :30301 --jsonrpc :10001 --log-level DEBUG --log-to ./log
```

Comment:
Here we have to do one of the following:
- add the flag and the value for all the validators
- to remove it and add `defaultSecretsConfigPath` when setting the flag

P.S. We can update the description of the flag, because the secrets are not used for the Hashicorp Vault only anymore.